src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -552,12 +552,11 @@ protected override void ProcessRecord()
                             using HttpResponseMessage response = GetResponse(client, request, handleRedirect);
 
                             string contentType = ContentHelper.GetContentType(response);
-                            string respVerboseMsg = string.Format(
-                                CultureInfo.CurrentCulture,
-                                WebCmdletStrings.WebResponseVerboseMsg,
-                                response.Content.Headers.ContentLength,
-                                contentType);
-
+                            long? byteLength = response.Content.Headers.ContentLength ?? null;
+                            string respVerboseMsg = byteLength is null ?
```

Comment:
```suggestion
                            string respVerboseMsg = response.Content.Headers.ContentLengthis null ?
```

Reply:
LGTM, thanks. (@iSazonov, a space is missing before `is`).